.prettierignore

```diff
@@ -61,3 +61,6 @@ eng/feeds/
 
 # Tm Grammars generated files
 grammars/
+
+# auto generated code model files
+packages/http-client-csharp/generator/TestProjects/**/tspCodeModel.json
```

Comment:
do we need to ignore the config file too?

Reply:
config file happens to be written in a good format so no, but if it becomes a problem later we can add it.